ql/src/test/queries/clientpositive/view_top_relnode_not_project_authorization.q

```diff
@@ -0,0 +1,16 @@
+set hive.security.authorization.enabled=true;
+create table t1 (username string, id int);
+
+create view vw_t1 as select distinct username from t1 limit 5;
+explain cbo select * from vw_t1;
+select * from vw_t1; 
+
+create view vw_t2 as 
```

Comment:
Does vw_t2 and vw_t3 have a different operator on top of the view? If not then we are not really adding test coverage so we should drop those test cases.

Reply:
Removed `vw_t3` as it was producing a `HiveSortLimit`, which is already covered in another test. Kept `vw_t2` as it produces a `HiveFilter` on top.